Ontology/Willow/Capability/State/Binary State/Alarm State/Pressure/StaticPressure/Refrigerant/LowCompressorEnteringRefrigerantStaticPressureAlarmState.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@id": "dtmi:com:willowinc:LowCompressorEnteringRefrigerantStaticPressureAlarmState;1",
+  "@type": "Interface",
+  "displayName": {
+    "en": "Low Compressor Entering Refrigerant Static Pressure Alarm State"
+  },
+  "description": {"en": "The Boolean state of the low compressor entering refrigerant static pressure alarm is 'alarm' when 'true' and 'normal' when 'false'"},
```

Comment:
The order of terms in the id, display name, and description seem jumbled from how you would likely say this example.